### PR TITLE
Document use of context in put stored script

### PR DESCRIPTION
**Pull request by jdconrad**
This documents how to test compile a stored script against a specific context when using PUT/POST.

### CODE REVIEW

File: docs/reference/scripting/using.asciidoc [hunk 1]
```diff
@@ -155,6 +155,21 @@ POST _scripts/calculate-score
 -----------------------------------
 // TEST[setup:twitter]
 
+Specify a context as part of the url path to test compile a stored script
```
Comment (rjernst) [inline docs/reference/scripting/using.asciidoc]: Maybe start with "You may also"

Comment (rjernst) [reply docs/reference/scripting/using.asciidoc]: Also, I would remove "test", since this is just about compiling, it does not run the script.

**Review by rjernst — APPROVED** [review]
Looks good, a couple suggestions.

File: docs/reference/scripting/using.asciidoc (hunk 1 above)
Comment (jdconrad) [reply docs/reference/scripting/using.asciidoc]: Done.

**Comment by jdconrad** [comment]
@rjernst Thanks for the review. Will commit as soon as CI passes.

**Pull request merged**